app/src/organisms/Devices/ProtocolRun/ProtocolRunRunTimeParameters.tsx

```diff
@@ -189,19 +197,20 @@ interface StyledTableRowProps {
 
 const StyledTableRow = styled.tr<StyledTableRowProps>`
   display: grid;
-  grid-template-columns: 1fr 1fr;
+  grid-template-columns: 0.35fr 0.35fr;
   grid-gap: 48px;
```

Comment:
could you also update this?

```suggestion
  grid-gap: SPACING.spacing48;
```